client/extensions/woocommerce/app/store-stats/constants.js

```diff
@@ -7,35 +7,35 @@ export const sparkWidgetList1 = [
 	{
 		key: 'products',
 		title: translate( 'Products Purchased' ),
-		type: 'number'
+		format: 'number'
 	},
 	{
 		key: 'avg_products_per_order',
 		title: translate( 'Products Per Order' ),
-		type: 'number'
+		format: 'number'
 	},
 	{
 		key: 'coupons',
 		title: translate( 'Coupons Used' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Coupons' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).